pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -633,8 +634,9 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
         # Check adjoint diff support
         self._check_adjdiff_supported_operations(tape.operations)
 
+        split_obs = self._dp.getTotalDevices() if self._batch_obs else False
```

Comment:
Any concerns here if this is hit on a node with MPI enabled? Should we allow the user to define the number instead, as we do with batch_obs?